pkg/controller/nodes/subworkflow/launchplan.go

```diff
@@ -69,15 +69,15 @@ func (l *launchPlanHandler) CheckLaunchPlanStatus(ctx context.Context, nCtx hand
 
 	if err != nil {
 		// THIS SHOULD NEVER HAPPEN
-		return handler.DoTransition(handler.TransitionTypeEphemeral, handler.PhaseInfoFailure(errors.RuntimeExecutionError, "failed to create unique ID", nil)), nil
+		return handler.DoTransition(handler.TransitionTypeEphemeral, handler.PhaseInfoFailure(core.ExecutionError_SYSTEM, errors.RuntimeExecutionError, "failed to create unique ID", nil)), nil
 	}
 
 	wfStatusClosure, err := l.launchPlan.GetStatus(ctx, childID)
 	if err != nil {
 		if launchplan.IsNotFound(err) { // NotFound
 			errorCode, _ := errors.GetErrorCode(err)
 			err = errors.Wrapf(errorCode, nCtx.NodeID(), err, "launch-plan not found")
-			return handler.DoTransition(handler.TransitionTypeEphemeral, handler.PhaseInfoFailure(errorCode, err.Error(), &handler.ExecutionInfo{
+			return handler.DoTransition(handler.TransitionTypeEphemeral, handler.PhaseInfoFailure(core.ExecutionError_USER, errorCode, err.Error(), &handler.ExecutionInfo{
```

Comment:
hmm is this a user error? if we come here we are saying the system guaranteed we launched something, right? it can't just disappear by any user action... not that we ever hit this error

Reply:
hmm but then this should not happen. It could be a system error where we have a corruption